### ISSUE TITLE
darker image for "unknowns" rotation

### ISSUE THREAD

**Issue by ValWood** [pombase/website#554]
Kim mentioned yesterday that darker images will probably look better (or at least more of a mixture). At presnet most of our backgrounds are "light"

The cytokinesis video has a dark background.

I remade the "unknowns rotation image with a dark grey background. (in drop box)

**Comment by ValWood** [comment]
I think it might be too dark, with the dark cytokinesis we are going a bit goth (do you remember  the phase when all web pages had a black background? )

Anyway I have done some called unknown_medium. I think these might work OK....

**Comment by ValWood** [comment]
Do you think we should use the  image I made with a darker background?
Can close this if you think the existing one is better.
v

**Comment by kimrutherford** [comment]
I'm not sure.  Here's what it looks like.  It should be visible if you visit the URL for my desktop: http://pombase2-kmr.bioinformatics.nz/

I'll try the medium background one you made next.

![screenshot from 2017-10-23 10-48-06](https://user-images.githubusercontent.com/90474/31866697-d502ba42-b7df-11e7-9d43-6f5b8264e471.png)

**Comment by ValWood** [comment]
I think the dark one is too dark with the contrasrt with the pie. I should have deleted that one. The medium one might work....

**Comment by kimrutherford** [comment]
I've swapped it in the dev version on my desktop so you can see it in context: http://pombase2-kmr.bioinformatics.nz/

Let me know if you think we should go with it.

**Comment by kimrutherford** [comment]
> but it looks OK now we have a dark one and a few others

Yep, and the infographic rather than the curation graph in the middle panel improves things.
